client/src/main/scala/org/http4s/client/oauth1/Header.scala

```diff
@@ -0,0 +1,77 @@
+package org.http4s.client.oauth1
+
+import cats.Show
+import cats.kernel.Order
+import cats.implicits._
+sealed trait Header {
+  val headerName: String
+  val headerValue: String
+  def toTuple: (String, String) =
+    headerName → encode(headerValue)
```

Comment:
Unicode arrows are deprecated in Scala 2.13.  Better to use `=>`.

Reply:
Done